### PR TITLE
custom properties docs updates

### PR DESCRIPTION
**Pull request by ajwootto**

### CODE REVIEW

**Review by cdruxerman — COMMENTED** [review]
Left a few comments

File: docs/essentials/targeting.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ title: Targeting
 sidebar_position: 8
 ---
 
-Targeting rules can be used to grant features to specific user groups, incrementally roll out features for monitoring, or create and test different feature configurations by using [multiple variables](/essentials/variables).
+Targeting rules can be used to grant features to specific user groups, 
+incrementally roll out features for monitoring, or create and test different feature configurations.
```
[the remark this answers is not in the document]
Comment (ajwootto) [reply docs/essentials/targeting.md:7]: to fit in the max width of a line in my editor. Markdown ignores single line breaks.

File: docs/essentials/targeting.md [hunk 2]
```diff
@@ -14,6 +15,47 @@ Be sure to check out our advanced targeting documentation which covers topics li
...
+that you set yourself in the SDK but are built in to the platform, and custom properties that you define to extend
+the built-in Targeting properties.
+
+Below is a summary of the properties built in to the platform, and how to specify them in the SDK:
+
+| Property Name    | Purpose                                                                                | How to Set               |
+|------------------|----------------------------------------------------------------------------------------|--------------------------|
+| User ID          | Unique identifier for this user. Also used for distribution and rollout randomization. | Set "user_id" property   |
+| User Email       | Email associated to this user                                                          | Set "email" property     |
+| App Version      | Version of the application currently in use.                                           | Automatically set by SDK |
```
[the remark this answers is not in the document]
Comment (ajwootto) [reply docs/essentials/targeting.md:31]: it seems like it's set automatically on mobile but is a property you can set yourself on JS. Another reason these properties are just confusing IMO

[the remark this answers is not in the document]
Comment (ajwootto) [reply docs/essentials/targeting.md:31]: this list also omits some of the properties that exist in our User types and are collected by the SDKs but don't exist on the targeting interface anymore, like appBuild, language etc.

**Comment by ajwootto** [comment]
* **#575** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/devcycle-docs/575?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ajwootto and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Pull request merged**